src/components/Tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Styles from './style';
+
+class Tooltip extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.setTooltipPosition = this.setTooltipPosition.bind(this);
+    this.state = {
+      childWidth: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.setTooltipPosition();
+    window.addEventListener("resize", this.setTooltipPosition);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener("resize", this.setTooltipPosition);
+  }
+
+  setTooltipPosition() {
+    // Getting the first child width to calculate Tooltip position
+    const childWidth = this.tooltipWrapper.children[0].children[0].getBoundingClientRect().width;
```

Comment:
2/2 @anafilipadealmeida because I have to add that div wrapping the children, is taking a full width, even if I'm surrounding an icon (30px width for example) with the Tooltip, causing the Tooltip to be centered relative to the div, instead of the actual icon. This is the reason I added this line, to get the actual child width.

Reply:
Thanks for adding this workaround for now @MayaUribe 💯! See above re: styled-components upgrade since it would be nice if we could remove this sort of tricky code.